website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -759,6 +759,10 @@ A `service_mesh_profile` block supports the following:
 
 -> **NOTE:** Currently only one Internal Ingress Gateway and one External Ingress Gateway are allowed per cluster
 
+* `revisions` - (Optional) Specify 1 or 2 Istio control plane revisions for managing minor upgrades using the canary upgrade process. For example, create the resource with `revisions` set to `["asm-1-20"]`, or leave it empty (the `revisions` will only be known after apply). To start the canary upgrade, change `revisions` to `["asm-1-20", "asm-1-21"]`. To roll back the canary upgrade, revert to `["asm-1-20"]`. To confirm the upgrade, change to `["asm-1-21"]`.
```

Comment:
This is a `Required` property in 4.0 which is why it has been listed further up on line 754. Can you remove this property description and move the note below further up?

Reply:
Thanks for spotting this, updated, I am using the new definition because I think it gives a clearer explanation. Let me know any other changes